src/Tasks/Microsoft.NET.Build.Tasks/AppHost.cs

```diff
@@ -17,18 +17,32 @@ public static class AppHost
         private const string _placeHolder = "c3ab8ff13720e8ad9047dd39466b3c8974e592c2fa383d4a3960714caef0c4f2"; //hash value embedded in default apphost executable
         private readonly static byte[] _bytesToSearch = Encoding.UTF8.GetBytes(_placeHolder);
 
+        /// <summary>
+        /// Settings to customize the apphost.
+        /// </summary>
+        public class HostCustomizationSettings
```

Comment:
Nit: I think "Customization" is redundant when we're already talking about "Settings" or "Options". I also think "Host" is redundant in nested scope with AppHost. 

Even more subjective, but I also prefer "Options" over "Settings".

And finally, Framework Design Guidelines discourage public nested classes. Those don't apply here, but I like to follow this one. 

I would prefer any of these (listed in my personal order of preference):

* `AppHostOptions` (top-level)
* `AppHostSettings` (top-level)
* `AppHost.Options` (nested)
* `AppHost.Settings` (nested)

Reply:
I thought about making the AppHostOptions class separate (top-level), but then it would require its own file and seems way too small for that. But I will happily oblige if others would like that.

Reply:
I don't mind a new file since we're likely to keep expanding the options as the 3.0 features get implemented.  I'd also second `AppHostOptions`.